build/windows/installer/conf/fluent-multiline.conf

```diff
@@ -0,0 +1,92 @@
+<source>
+  type heartbeat_request
+  run_interval 30m
+  @log_level info
+</source>
+
+<source>
+  @type cadvisor_perf
+  tag oms.api.cadvisorperf
+  run_interval 60
+  @log_level debug
+</source>
+
+<source>
```

Comment:
if I understand correctly, with multi-line, we will be using Fluent bit tail plugin not the Fluent-D,right? in that case, why do we need FluentD tail plugin?